frameworks/Crystal/kemal/server-postgres.cr

```diff
@@ -0,0 +1,147 @@
+require "kemal"
+require "pg"
+require "pool/connection"
+require "html_builder"
+
+# Compose Objects (like Hash) to have a to_json method
+require "json/to_json"
+
+DB = ConnectionPool.new(capacity: 25, timeout: 0.01) do
+  PG.connect("postgres://benchmarkdbuser:benchmarkdbpass@#{ENV["DBHOST"]? || "127.0.0.1"}/hello_world")
+end
+
+class CONTENT
+  UTF8 = "; charset=UTF-8"
+  JSON = "application/json"
+  PLAIN = "text/plain"
+  HTML = "text/html" + UTF8
+end
+
+ID_MAXIMUM = 10_000
+
+private def randomWorld
+  id = rand(1..ID_MAXIMUM)
+  result = DB.connection.exec({Int32, Int32}, "SELECT id, randomNumber FROM world WHERE id = $1", [id]).rows.first
```

Comment:
This only creates one connection and reuses that. 

You can paralellize this query with something like these.

```crystal
conn = DB.checkout
result = conn.exec({Int32, Int32}, "SELECT id, randomNumber FROM world WHERE id = $1", [id]).rows.first
DB.checkin(conn)
```